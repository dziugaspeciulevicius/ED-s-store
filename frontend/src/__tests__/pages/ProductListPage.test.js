import React from "react";
import { shallow, mount, render } from "enzyme";
import "../../setupTests";
import ProductListPage from "../../pages/ProductListPage.component"
jest.mock("../../pages/ProductListPage.component");

describe("<ProductListPage />", () => {
    it("Renders <ProductListPage /> component correctly", () => {
      // shallow creates instance of a component
      expect(shallow(<ProductListPage />)).toMatchSnapshot();
    });
});