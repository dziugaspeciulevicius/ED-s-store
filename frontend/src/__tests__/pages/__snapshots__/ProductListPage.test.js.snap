// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductListPage /> Renders <ProductListPage /> component correctly 1`] = `ShallowWrapper {}`;
